tools/cosmovisor/cmd/cosmovisor/help_test.go

```diff
@@ -15,79 +10,6 @@ type HelpTestSuite struct {
 	suite.Suite
 }
 
-func TestHelpTestSuite(t *testing.T) {
```

Comment:
I think this one is still needed to run the whole suite.

Reply:
I'll bring it back

Reply:
lol I thought I was reading my own PR, cause, linting